src/Migrations/Version20191125205634.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Verta nepalikinėti šiukšlių.
Jau yra viena migracija `Version20191125200535`, kuri prideda laukelius. 

Papildomi failai yra tik triukšmas

Reply:
Kodas veikia :heavy_plus_sign: buvo atskira pasakita (gal ir be namų darbų) apie `Doctrine` :heavy_plus_sign: vėlai atrašau (jau greitai final atsiskaitymas) – tai už namų darbus balo nemažinu – palieku tavo sąžinei, ar verta pasigilinti